### DISCUSSION TITLE
samples https://django-formset.fly.dev/bootstrap/

### DISCUSSION THREAD

**Discussion by richard7III** [jrief/django-formset#166] [General]
Thanks for a great package! 
While having already used in a project, still struggle with styling. Particularly with Select field.
Can you please somebody point me to source code for samples https://django-formset.fly.dev/bootstrap/ ?

Cheers!

**Comment by jrief** [comment]
This example https://django-formset.fly.dev/bootstrap/country uses the Selectize-widget. The code for the example can be found in /testapp/.

django-formset adopts itself to the styling of the underlying framework. It pilfers the styles from the widgets to be replaced. In the above case this is the `<select>…</select>`-widget.

**Comment by richard7III** [comment]
Thanks Jacob!

Yes, I am using Selectize-widget:

lass BeneficiarydetailsForm(forms.Form):
default_renderer = FormRenderer(
form_css_classes='row',
field_css_classes={
'*': 'mb-2 col-12',
'currency': 'mb-2 col-4',
'beneficiary_nickname': 'mb-2 col-8'
},
)
#Beneficiary Nickname
beneficiary_nickname = CharField(label='Beneficiary nickname',
widget=TextInput(attrs={'placeholder': 'Beneficiary nickname'}))
#Beneficiary Country
beneficiary_country = ModelChoiceField(
queryset= Country.objects.all(),
label='Beneficiary Country',
widget=Selectize(
search_lookup='name__icontains',
placeholder="Select Country",
),
)

and in template:

<django-formset endpoint="{{ request.path }}" csrf-token="{{ csrf_token }}">
{% render_form form "bootstrap" %}
<div class="offset-sm-3">
<button type="button" df-click="submit -> proceed">Submit</button>
</div>
</django-formset>

, but the style does not affect SELECT part:

[image: image.png]

...even it does style TextInput widget..

What am I missing?

Cheers!

**Reply by jrief** [reply]
Could you please format the code and send me a screenshot and explain what iyo should look different.
